### PR TITLE
This one- add delete Inventory Item capability in the backend 

### PR DESCRIPTION
**Pull request by sanchezfernandes**
-Deletes a single item
-Updates status code
-Also changed the route from "Inventory" to "Inventories"

### CODE REVIEW

**Review by trumerfe — APPROVED** [review]
clean, ty for fixing inventories endpoint

File: server.js [hunk 1]
```diff
@@ -15,10 +15,10 @@ app.use(express.json());
 
 //basic routes for the APIs
 app.use('/warehouses', warehouseRoutes);
-app.use('/inventory', inventoryRoutes);
-app.use('/', (_req, res)=>{
+app.use('/inventories', inventoryRoutes);
```
Comment (Edthev) [inline server.js:18]: beautiful 👍👍

**Review by Edthev — APPROVED** [review]
Approved

**Review by jwuzzle — APPROVED** [review]
Very nice!

**Pull request merged**